go/worker/compute/executor/committee/prune.go

```diff
@@ -18,8 +18,8 @@ func (p *pruneHandler) Prune(rounds []uint64) error {
 	height := p.commonNode.CurrentBlockHeight
 	p.commonNode.CrossNode.Unlock()
 
-	// Make sure we never prune past the last successful round as we need that round in history so
-	// we can fetch any needed round results.
+	// Make sure we never prune past the last normal round, as some runtimes will do historic queries
+	// for things that are not available in last consensus state.
```

Comment:
```suggestion
	// for things that are not available in the last consensus state (e.g. delegation/undelegation
	// events that happened while the runtime was suspended or not producing blocks).
```
I would keep the example, is informative.